Mathlib/Analysis/Convex/Cone/Caratheodory.lean

```diff
@@ -0,0 +1,259 @@
+/-
+Copyright (c) 2023 Apurva Nakade. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Apurva Nakade
+-/
+import Mathlib.Analysis.Convex.Cone.Pointed
+import Mathlib.Analysis.Convex.Caratheodory
+
+/-!
+# Carathéodory's convexity theorem for convex cones
+
+In this file we prove Carathéodory's convexity theorem for convex cones. Carathéodory's theorem
+states that if `x` is in the convex cone generated by a set `s`, then `x` is a convex combination
+of linearly independent elements of `s`. This is the replacement of the usual statement of vector
+spaces having bases for the setting of convex cones.
+
+The proof follows the proof of Carathéodory's theorem for convex hulls closely.
+
+
+## Main results
+
+- `toPointedCone_eq_union`: Carathéodory's convexity theorem for convex cones.
+- `mem_toPointedCone_iff_eq_pos_convex_span`: A more explicit version of `toPointedCone_eq_union`.
+
+-/
+
+
+universe u
+
+open Set Finset
```

Comment:
Why are these two on the same line but `BigOperators` on a separate line?

Reply:
Oops, I'll fix this. Thanks!